pkg/api/admin/openshiftcluster.go

```diff
@@ -71,6 +71,7 @@ type ClusterProfile struct {
 	Domain          string `json:"domain,omitempty"`
 	Version         string `json:"version,omitempty"`
 	ResourceGroupID string `json:"resourceGroupId,omitempty"`
+	FipsEnabled     bool   `json:"fipsEnabled,omitempty"`
```

Comment:
There is not bools in API. Make it tristate variable/custom type. Example:
```
// EncryptionAtHost represents encryption at host state
type EncryptionAtHost string

// EncryptionAtHost constants
const (
	EncryptionAtHostEnabled  EncryptionAtHost = "Enabled"
	EncryptionAtHostDisabled EncryptionAtHost = "Disabled"
)
```
This is msft api requirement.

Reply:
removed cli and changed fips flag to mirror pattern above